modules/test/playwright/tests/portal-workflow-kaleo-designer-web/main/workflowDocumentsAndMedia.spec.ts

```diff
@@ -0,0 +1,190 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../../fixtures/dataApiHelpersTest';
+import {documentLibraryPagesTest} from '../../../fixtures/documentLibraryPages.fixtures';
+import {loginTest} from '../../../fixtures/loginTest';
+import {workflowPagesTest} from '../../../fixtures/workflowPagesTest';
+import {createCategories} from '../../../helpers/CreateCategories';
+import getGlobalSiteId from '../../../utils/getGlobalSiteId';
+import {assetCategoriesPagesTest} from '../../asset-categories-admin-web/main/fixtures/assetCategoriesAdminPagesTest';
+import {cmsPagesTest} from '../../site-cms-site-initializer/main/fixtures/cmsPagesTest';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	assetCategoriesPagesTest,
+	cmsPagesTest,
+	documentLibraryPagesTest,
+	workflowPagesTest,
+	loginTest()
+);
+
+test.afterEach(async ({documentLibraryPage, page}) => {
+	await test.step('Cleanup: delete folder', async () => {
+		await documentLibraryPage.goto();
+
+		const folderLocator = page.locator(
+			`.card-body:has-text('DM Folder Name')`
+		);
+
+		if (await folderLocator.isVisible()) {
+			await documentLibraryPage.deleteFolder('DM Folder Name');
+		}
+	});
+});
+
+test(
+	'save as draft and view categories',
+	{tag: '@LPD-53559'},
+	async ({
+		apiHelpers,
+		documentLibraryEditFilePage,
+		documentLibraryEditFolderPage,
+		documentLibraryPage,
+		page,
+		workflowTasksPage,
+	}) => {
+		await test.step('Create Categories', async () => {
+			const categoryNames = [{name: 'legal'}, {name: 'other'}];
+
+			const categories: Array<any> = await createCategories({
```

Comment:
Typescript has already made this `Array<any>` inference:

![image](https://github.com/user-attachments/assets/acb4feac-d5fc-48b2-9272-e83a08b17458)

Reply:
This is because we are omitting the `vocabularyId` when creating the type.

`export type TCategory = Omit<postTaxonomyVocabularyTaxonomyCategoryProps, 'vocabularyId'>;`

In general it is not a good approach to use any, I always try to have the mindset of keeping my typings strong to take full advantage of the advantages that TypeScript offers.